src/main/java/calculator/Accumulator/Accumulator.java

```diff
@@ -0,0 +1,20 @@
+package calculator.Accumulator;
+
+public class Accumulator {
+    public static final String ADD_SYMBOL = "+";
+    public static final String SUB_SYMBOL = "-";
+    public static final String MULTIPLY_SYMBOL = "*";
+    public static final String DIVISION_SYMBOL = "/";
```

Comment:
서로 관련성이 있는 상수는 enum으로 구현해 보면 어떨까?


```suggestion
    PLUS("+") {
        @Override
        public int calculate(int operand1, int operand2) {
            return operand1 + operand2;
        }
    }
```

enum으로 구현한다면 위와 같이 사칙 연산 단위에 해당하는 메서드를 연결할 수도 있다.
꼭 enum이 아니라 인퍼페이스를 만들고, 인터페이스에 대한 구현체를 만드는 방식으로 도전해 보는 것도 좋은 접근 방식이다.